File: contrib/libatscc2py/CATS/bool_cats.py
######
#
# HX-2014-08:
# for Python code translated from ATS
#
######

######
#beg of [bool_cats.py]
######

######
from ats2pypre_basics_cats import *
######

############################################
#
def ats2pypre_not_bool0(x): return (not(x))
def ats2pypre_not_bool1(x): return (not(x))
#
############################################
#
def ats2pypre_print_bool(b):
  return ats2pypre_fprint_bool(sys.__stdout__, b)
def ats2pypre_prerr_bool(b):
  return ats2pypre_fprint_bool(sys.__stderr__, b)
#
def ats2pypre_fprint_bool(out, b): return ats2pypre_fprint_obj(out, b)
#
############################################

###### end of [bool_cats.py] ######

File: contrib/libatscc/libatscc2pl/CATS/bool_cats.pl
######
#
# HX-2014-11:
# for Perl code translated from ATS
#
######

######
#beg of [bool_cats.pl]
######

############################################
#
sub
ats2plpre_print_bool($)
{
  if($_[0]) { print STDOUT "true"; } else { print STDOUT "false"; }; return;
}
sub
ats2plpre_prerr_bool($)
{
  if($_[0]) { print STDERR "true"; } else { print STDERR "false"; }; return;
}
#
############################################

######
1; #note that it is needed by 'use' or 'require'
######

######
#end of [bool_cats.pl]
######

